File: Java-ProyectUberClass/uberBlack.java
import java.util.ArrayList;
import java.util.Map;

public class uberBlack extends car{
    Map<String, ArrayList<String>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public uberBlack(String license, account driver, Map<String, ArrayList<String>> typeCarAccepted, ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: PHP-ProyectUberClass/uberBlack.php
<?PHP
require_once('car.php');
class uberBlack extends car{
    public $typeCarAccepted;
    public $seatsMaterial;
    public function __construct($typeCarAccepted, $seatsMaterial, $license, $driver)
    {
        parent::__construct($license, $driver);
        $this->typeCarAccepted->$typeCarAccepted;
        $this->seatsMaterial->$seatsMaterial;    
    }
}

?>